src/api/addresses.ts

```diff
@@ -54,13 +76,38 @@ export const fetchLedgerLiveAddresses = async (n = 10): Promise<string[]> => {
             startPath: [
               HARDENED_OFFSET + 49,
               HARDENED_OFFSET + 60,
-              HARDENED_OFFSET,
-              i,
+              HARDENED_OFFSET + i + startPathIndex > 0 ? startPathIndex : 0,
```

Comment:
Can this not be handled in `getStartPath` with a `BTC_LEDGER_LIVE_DERIVATION`? It seems like you could change `getStartPath` to include a switch statement for bespoke path formats like this one

Reply:
I updated the function to accept a path, a value to increment the default value, and which value in the path to increment. It should be able to handle any derivation scheme.